src/Data/Vec/Properties.agda

```diff
@@ -1025,17 +1039,32 @@ map-reverse f (x ∷ xs) = begin
   where open ≡-Reasoning
 
 -- append and reverse
+toList-∷ʳ : ∀ x (xs : Vec A n) → toList (xs ∷ʳ x) ≡ toList xs List.++ List.[ x ]
+toList-∷ʳ x []       = refl
+toList-∷ʳ x (y ∷ xs) = cong (y List.∷_) (toList-∷ʳ x xs)
+
+toList-reverse : ∀ (xs : Vec A n) → toList (reverse xs) ≡ List.reverse (toList xs)
+toList-reverse [] = refl
+toList-reverse (x ∷ xs) = begin
+  toList (reverse (x ∷ xs))          ≡⟨ cong toList (reverse-∷ x xs) ⟩
```

Comment:
Can we align the reasoning combinators? (you may need to install a monospaced font to make this easier)